src/main/java/com/altenheim/kalender/controller/logicController/AppointmentEntryFactory.java

```diff
@@ -1,21 +1,71 @@
 package com.altenheim.kalender.controller.logicController;
 
+import com.altenheim.kalender.TempTestClasses.CreateDummyEntries;
+import com.altenheim.kalender.TempTestClasses.ICreateDummyEntries;
+import com.altenheim.kalender.controller.viewController.CalendarViewOverride;
 import com.altenheim.kalender.interfaces.IAppointmentEntryFactory;
+import com.altenheim.kalender.interfaces.ICalendarEntriesModel;
+import com.altenheim.kalender.models.ContactModel;
+
 import java.time.LocalDate;
 import java.time.LocalTime;
+import java.time.ZoneId;
+import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
 import java.util.concurrent.ThreadLocalRandom;
 import com.calendarfx.model.Calendar;
+import com.calendarfx.model.CalendarSource;
 import com.calendarfx.model.Entry;
 
 
-public class AppointmentEntryFactory implements IAppointmentEntryFactory
+public class AppointmentEntryFactory extends CreateDummyEntries implements IAppointmentEntryFactory
 {    
+    private ICalendarEntriesModel allCalendars;
+    private CalendarViewOverride calendarView;
+
```

Comment:
Übergabe der Sammlung aller Kalender, weil die Factory hier reinschreibt. Zur Zeit noch mit dem Zufallsgenerator, das wird natürlich später ersetzt, so dass, wenn Einträge gemacht werden hier gesteuert wird in welchen Kalender sie geschrieben werden und anschließend gespeichert wird. Die View wird übergeben um bei Update zu rendern, das ist nicht so richtig sauber und gehört hier eigentlich nicht hin sondern in den entsprechenden ViewController. Bzw. eigentlich müsste hier ein Listener regisitriert sein,  so dass beim Update der Collections ein Renderupdate erfolgt - glaube dafür hat CalenderFX gute Möglichkeiten. Aber erstmal ist es für das Sprintziel und den Einbau der Funktionen  ne pragmatische Lösung.